iap/README.md

```diff
@@ -76,6 +76,46 @@ service account private key can impersonate that account!
    ```
 2. Copy `validate_jwt.py` into your application.
 
+## Using generate_self_signed_jwt
+
+### Self-signed JWT with IAM Credentials API
+
+Ensure that you are in the correct working directory: (/python-docs-samples/iap):
+
+1. Install the libraries listed in `/python-docs-samples/iap/requirements.txt`, e.g. by running:
+
+```
+   virtualenv/bin/pip install -r requirements.txt
+```
+
+2. Call `sign_jwt` in the python file. This example would create a JWT for the service account email@gmail.com to access the IAP protected application hosted at https://example.com.
```

Comment:
You can just use `1.` for each entry rather than just numbers, Markdown will figure out the sequence for you. That way, they never go out of sync.